sdk/arm/resources/2020-06-01/armresources/README.md

```diff
@@ -0,0 +1,48 @@
+# Azure Resources Module for Go
+
+[![PkgGoDev](https://pkg.go.dev/badge/github.com/Azure/azure-sdk-for-go/sdk/arm/resources/2020-06-01/armresources)](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/arm/resources/2020-06-01/armresources)
+
+The `armresources` module provides operations for working with resources and resource groups.
```

Comment:
Should this say "for working with Azure resources and resource groups"?

Reply:
We can.  For reference I copied this verbatim from the swagger.

Reply:
I see. I think it's better with Azure in there, but if the swagger is like that then I dont feel strongly enough to push on it.

Reply:
I added it.